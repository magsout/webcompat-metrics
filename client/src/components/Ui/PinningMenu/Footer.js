/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import SVGInline from "react-svg-inline";

import SVGTwitter from "../../../assets/svg/iconmonstr-twitter-1.svg";
import SVGGithub from "../../../assets/svg/iconmonstr-github-1.svg";

import classes from "./styles.css";

const Footer = props => {
  return (
    <footer className={classes.footer}>
      <div className={classes.socials}>
        <a
          className={classes.social}
          href="https://twitter.com/webcompat"
          title="Twitter webcompat accounts"
          target="_blank"
          rel="noopener noreferrer">
          <SVGInline
            svg={SVGTwitter}
            fill="#1da1f2"
            height={"1.5em"}
            cleanup={[
              "title",
              "desc",
              "comment",
              "defs",
              "sketchMSShape",
              "sketchMSPage",
              "sketchMSLayerGroup",
            ]}
          />
        </a>
        <a
          className={classes.social}
          href="https://github.com/webcompat"
          title="Github webcompat account"
          target="_blank"
          rel="noopener noreferrer">
          <SVGInline
            svg={SVGGithub}
            fill="#c6cbd1"
            height={"1.5em"}
            cleanup={[
              "title",
              "desc",
              "comment",
              "defs",
              "sketchMSShape",
              "sketchMSPage",
              "sketchMSLayerGroup",
            ]}
          />
        </a>
      </div>
      <div className={classes.footerText}>
        <span>Created and maintained by Webcompat</span>
      </div>
    </footer>
  );
};

export default Footer;
