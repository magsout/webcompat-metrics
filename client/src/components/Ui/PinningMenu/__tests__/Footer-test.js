/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import Footer from "../Footer";

it("renders Footer default correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
