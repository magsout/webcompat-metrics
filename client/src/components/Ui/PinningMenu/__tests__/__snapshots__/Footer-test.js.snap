// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer default correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="socials"
  >
    <a
      className="social"
      href="https://twitter.com/webcompat"
      rel="noopener noreferrer"
      target="_blank"
      title="Twitter webcompat accounts"
    >
      <span
        className="SVGInline SVGInline--cleaned"
        dangerouslySetInnerHTML={
          Object {
            "__html": "test-file-stub",
          }
        }
      />
    </a>
    <a
      className="social"
      href="https://github.com/webcompat"
      rel="noopener noreferrer"
      target="_blank"
      title="Github webcompat account"
    >
      <span
        className="SVGInline SVGInline--cleaned"
        dangerouslySetInnerHTML={
          Object {
            "__html": "test-file-stub",
          }
        }
      />
    </a>
  </div>
  <div
    className="footerText"
  >
    <span>
      Created and maintained by Webcompat
    </span>
  </div>
</footer>
`;
